ts_src/psbt.ts

```diff
@@ -977,6 +983,10 @@ class PsbtTransaction implements ITransaction {
     this.tx.addOutput(script, value, asset, nonce);
   }
 
+  addIssuance(args: AddIssuanceArgs): void {
```

Comment:
In this way, the partial outputs of the Psbt class would not be the same as those of the inner unsigned transaction.

tx.AddIssuance is adding outputs to the unsigned transaction, but no partial outputs are added to the current partial transaction!

I don't think the Transaction class needs a addIssuance method at all. Basically the current tx.addIssuance function has to become a method of the Psbt where you can read/write data from the underlying unsigned tx if you need (for generating entropy and then set the issuance object for input), and add outputs via the Psbt addOutput method.